### ISSUE TITLE
Consume: Amount on stock smaller then standard

### ISSUE THREAD

**Issue by 0xD0M1M0** [patzly/grocy-android#357]
Alpha v3:
I have a product with standard 500 grams and add manually via a purchase a product with 200 grams. The next time I scan the product to consume it, the field fills itself with the standard 500 grams and a warning appears that it has to be 200 grams or lower. Could the field fill itself with the maximum allowed value to make the workflow smoother?

**Comment by MrJohnZoidberg** [comment]
Could you explain what you mean with "standard amount" of a product, please? I cannot find a property with that name...

**Comment by 0xD0M1M0** [comment]
The product is configured as:
Quantity unit stock: grams
Default quantity unit purchase: can
Factor purchase to stock quantity unit: 500

Barcode:
Quantity unit: grams
Amount: 500

And I just verfied that it uses the amount from the barcode. So it's more a user error than a bug, but still if I have less on stock than the quantity attached to the barcode, I would prefer that it uses the current on stock value rather than the barcode amount to make the workflow faster.

**Comment by Saturo78** [comment]
Shouldn´t this also work if the amount in the selected location is fewer than 1?
If i change the location it does not.

**Comment by MrJohnZoidberg** [comment]
Do you mean the case where the preselected location contains less than the barcode amount or default amount?

**Comment by Saturo78** [comment]
It could be preselected or selected by user. If the amount is lower than 1, this amount should be preset otherwise you get the message that you have to lower the amount.

**Comment by MrJohnZoidberg** [comment]
@Saturo78 I don't understand exactly what you mean... Is the behavior which you think would be right also on the server interface?

**Comment by Saturo78** [comment]
When i have 0.7 on stock it makes no sense when the consume dialog is preset with 1, right?

**Comment by MrJohnZoidberg** [comment]
Yes now I understand and can reproduce, but I would prefer that the app has the same behavior like the web interface - there is always the default amount for consume prefilled on the server. Maybe you can create an issue there (if it doesn't exist) and then I would apply this on our app too...

**Comment by Saturo78** [comment]
If I'm honest, I have to say that I haven't used Grocy for quite a while. The app was just too slow for my large number of recipes and some processes were just too cumbersome. 

I also found the developer's handling of suggestions not always pleasant, so I feel rather less motivated to make further suggestions. 

My hope so far is that Grocy Android solves the points that the author of Grocy can not, or does not want to solve. I am now back to using the Android app I used before Grocy.
